python/lsst/pipe/drivers/coaddDriver.py

```diff
@@ -89,6 +93,11 @@ def __init__(self, reuse=tuple(), **kwargs):
         self.makeSubtask("assembleCoadd")
         self.makeSubtask("detectCoaddSources")
 
+    def __reduce__(self):
+        """Pickler"""
+        return unpickle, (self.__class__, [], dict(config=self.config, name=self._name,
+                                              parentTask=self._parentTask, log=self.log))
```

Comment:
Where's `reuse`?

Reply:
Thanks for the catch. Adding it to MultiBandDriver too.